doc/nrf/app_dev/device_guides/fem/fem_nrf21540_gpio.rst

```diff
@@ -74,3 +74,44 @@ To use nRF21540 in GPIO mode, complete the following steps:
       };
 
    The pins defined in the GPIO forwarder node in the application core's devicetree file must match the pins defined in the FEM nodes in the network core's devicetree file.
+
+#. On nRF54L devices, the GPIO pins of the SoC selected to control ``tx-en-gpios``, ``rx-en-gpios`` and ``pdn-gpios`` must support GPIOTE.
+   For example, on the nRF54L15 device, only pins belonging to GPIO P1 or GPIO P0 can be used and GPIO P2 pins cannot be used due to lack of related GPIOTE peripheral.
+   It is recommended for mentioned purpose to use these GPIO pins that belong to the PERI Power Domain of the nRF54L device.
```

Comment:
```suggestion
   It is recommended to use the GPIO pins that belong to the PERI Power Domain of the nRF54L device.
```

Reply:
fixed in df5396212ad429e6a139518bf499d78137e5a911